File: PokemonApp/app/src/main/java/com/example/pokemonmanagerAdix/MainActivity.java
package com.example.pokemonmanagerAdix;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.Button;
import android.view.View;
import android.content.Intent;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button btnPokemon1 = findViewById(R.id.btnPokemon1);
        Button btnPokemon2 = findViewById(R.id.btnPokemon2);

        btnPokemon1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, Pokemon1Activity.class);
                startActivity(intent);
            }
        });

        btnPokemon2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, Pokemon2Activity.class);
                startActivity(intent);
            }
        });
    }
}

File: MeusPokemon/app/src/main/java/com/example/meuspokemon/MainActivity.kt
package com.example.meuspokemon

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.Window
import android.view.WindowManager
import android.widget.Button

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setContentView(R.layout.activity_main)

        val bt_inventario = findViewById<Button>(R.id.bt_inventario)
        val bt_pokemon = findViewById<Button>(R.id.bt_pokemon)


        bt_inventario.setOnClickListener {
            startActivity(Intent(this, PagInventario::class.java))
        }

        bt_pokemon.setOnClickListener {
            startActivity(Intent(this, PagPokemon::class.java))
        }


    }
}